docs/testing/integration-testing.md

```diff
@@ -0,0 +1,79 @@
+---
+title: Integration testing
+category: Testing
+order: 5
+---
+
+# Integration testing
```

Comment:
I would not call these integration tests. Its rather something like read-world component testing

Reply:
good idea, that's more clear